### ISSUE TITLE
Multiple adresses

### ISSUE THREAD

**Issue by jahruhn** [cyberjunky/home-assistant-hvcgroep#13]
Is it also possible to work with multiple adresses? And how would I do this? I have tried adding another one with hvcgroup2. But it did not give me other entities to work with unfortunately? For 1 adress it works great btw, thank you!

**Comment by cyberjunky** [comment]
You catched a bug...
Will try to fix it!

```
Platform hvcgroep does not generate unique IDs. ID HVC Groep 5 already exists - ignoring sensor.hvc_groep_groene_bak_gft
Platform hvcgroep does not generate unique IDs. ID HVC Groep 6 already exists - ignoring sensor.hvc_groep_plastic_en_verpakking
Platform hvcgroep does not generate unique IDs. ID HVC Groep 3 already exists - ignoring sensor.hvc_groep_blauwe_bak_papier
Platform hvcgroep does not generate unique IDs. ID HVC Groep 2 already exists - ignoring sensor.hvc_groep_grijze_bak_restafval
````

**Comment by cyberjunky** [comment]
Fixed in version 1.0.11, see README, you need to delete the old entries manually.
